ubuntu/ubuntu-18.x/ubuntu-18.04-hpc/install_utils.sh

```diff
@@ -26,49 +26,5 @@ apt-get install -y libgirepository1.0-dev
 python3.8 -m pip install --ignore-installed PyGObject
 apt-get install -y software-properties-common
 
-apt-get -y install build-essential
-apt-get -y install numactl \
-                   rpm \
-                   libnuma-dev \
-                   libmpc-dev \
-                   libmpfr-dev \
-                   libxml2-dev \
-                   m4 \
-                   byacc \
-                   python-dev \
-                   python-setuptools \
-                   tcl \
-                   environment-modules \
-                   tk \
-                   texinfo \
-                   libudev-dev \
-                   binutils \
-                   binutils-dev \
-                   selinux-policy-dev \
-                   flex \
-                   libnl-3-dev \
-                   libnl-route-3-dev \
-                   libnl-3-200 \
-                   bison \
-                   libnl-route-3-200 \
-                   gfortran \
-                   cmake \
-                   libnl-3-dev \
-                   libnl-route-3-dev \
-                   libsecret-1-0 \
-		   ansible \
-                   dkms \
-                   jq
-
-# Install azcopy tool 
-# To copy blobs or files to or from a storage account.
-wget https://azcopyvnextrelease.blob.core.windows.net/release20210920/azcopy_linux_se_amd64_10.12.2.tar.gz
-tar -xvf azcopy_linux_se_amd64_10.12.2.tar.gz
-
-# copy the azcopy to the bin path
-pushd azcopy_linux_se_amd64_10.12.2
-cp azcopy /usr/bin/
-popd
-
-# Allow execute permissions
-chmod +x /usr/bin/azcopy
+#install apt pckages
+$UBUNTU_COMMON_DIR/instal_utils.sh
```

Comment:
apt-get update run twice in this file
Line 13 and again in common install_utils.sh. Please correct.

Reply:
The line 13 apt-get update needs to be there, so I could resolve this by moving the apt-get update out of the common file. But this will mean I need to have an install_utils.sh file for 18.04-LTS that just calls apt-get update and then calls the common install_utils.sh file. Is this alright?

Reply:
The apt-get update is used for microsoft.prod repo to be available in the list of repos. 
Is python 3.8 also dependent on that? if so, we could move the python installation steps after install_utils

Reply:
It looks like we can flip the order so I will do that.